herd/tests/instructions/AArch64/L005.litmus.expected-failure

```diff
@@ -0,0 +1 @@
+Warning: File "./herd/tests/instructions/AArch64/L003.litmus", line 7, characters 22-24: unexpected 'X0' (in prog) (User error)
```

Comment:
Hi, i guess the filename above has to be changed to `/herd/tests/instructions/AArch64/L005.litmus`.

Reply:
good spot, fixed :)